src/pudl_archiver/metadata/sources.py

```diff
@@ -287,4 +287,115 @@
+        "title": "DOE -- IRA Energy Community Data Layers",
+        "path": "https://edx.netl.doe.gov/dataset/ira-energy-community-data-layers",
+        "description": (
+            "An IRA Energy Community refers to areas designated under the U.S. Inflation Reduction "
+            "Act (IRA) as eligible for special incentives to promote clean energy development. These "
+            "communities are typically regions impacted by the transition away from fossil fuels, "
+            "such as those with closed coal mines or retired coal-fired power plants. The designation "
+            "aims to boost local economies and job creation by attracting renewable energy projects "
+            "like wind, solar, and battery storage, often offering enhanced tax credits to developers "
+            "working in these areas."
+            "This source contains data for two types of potentially qualifying energy communities: 1) "
+            "Census tracts and directly adjoining tracts that have had coal mine closures since 1999 "
+            "or coal-fired electric generating unit retirements since 2009. These census tracts qualify "
+            "as energy communities. 2) Metropolitan statistical areas (MSAs) and non-metropolitan statistical "
+            "areas (non-MSAs) that are energy communities for 2023 and 2024, along with their fossil "
+            "fuel employment (FFE) status."
+        ),
+        "working_partitions": {"years": [2024]},
+        "keywords": sorted(
+            {
+                "energy community",
+                "coal",
+                "mines",
+                "closure",
+                "ira",
+            }
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
+    "epadcejp": {
+        "title": "EPA -- Disadvantaged Community Energy Justice Program",
+        "path": "https://www.arcgis.com/home/webmap/viewer.html?url=https://geopub.epa.gov/arcgis/rest/services/ejscreen/environmental_climate_justice_program/MapServer&source=sd",
+        "description": (
+            "This dataset denotes whether a region is considered a disadvantaged community based on "
+            "the EPA's Energy Justice Program. An EPA Disadvantaged Community under the Energy Justice "
+            "Program is identified based on four key factors: environmental challenges like high pollution "
+            "exposure, economic hardships such as low income or high unemployment, social vulnerabilities "
+            "including health disparities and minority status, and geographic disadvantages in rural, urban, "
+            "or remote areas. These criteria ensure that support is directed to communities facing systemic "
+            "inequities and the greatest need for clean energy and environmental investments."
+        ),
+        "working_partitions": {"years": [2023]},  # not 100% sure on year
```

Comment:
This is a pretty new dataset that probably doesn't have any update frequency so years is probably not the right partition. But it's spatial so maybe by state? Depends how the data is available on the MapServer API.

Reply:
We're not even sure whether we can download this data tbh so maybe best to just add a comment saying not sure what the partitions are and leave it at that

Reply:
And whoever works on this dataset can update it.

Reply:
Yeah, Null partitions is probably fine. I'm pretty sure you can pull stuff from the MapServer -- it's a very friendly API IIRC.